ome/figure
# Shape edit 2

Some more copy, paste and selection functionality for ROIs.

To test:
 - Select a panel (with shapes - or add shapes to panel) and try to select multiple shapes by dragging a selection area.
 - Select All shapes via Cmd-A keyboard shortcut or via "Edit" menu.
 - The toolbar colour and line-thickness should update accordingly - E.g. if all selected shapes are same colour - that will show on toolbar. If not - will be white.
 - Updating colour or line-width in toolbar should apply to all selected shapes.
 - Copy multiple shapes to clipboard
 - Close the "Draw ROIs" dialog (with or without saving)
 - Select one or more other panels and Paste (Cmd V) to paste shapes onto the selected panels.

NB: Just noticed that I need to update the page "Edit > Paste" option to be enabled when we have copied shapes to the clipboard AND a panel is selected.
